src/dscanner/analysis/run.d

```diff
@@ -612,3 +619,37 @@ MessageSet analyze(string fileName, const Module m, const StaticAnalysisConfig a
 
 	return set;
 }
+
+MessageSet analyzeDmd(string fileName)
+{
+	import dmd.parse;
+	import dmd.astbase;
+	import dmd.id;
+	import dmd.globals;
+	import dmd.identifier;
+	import std.file : readText;
+	import dscanner.utils;
```

Comment:
You should use selective imports here. That is one of the reason the tests are crying.